examples/basic/basic.tf

```diff
@@ -4,6 +4,7 @@ module "ocean-gcp-k8s" {
 
   cluster_name = var.cluster_name
   location     = var.location
+  is_enabled   = var.is_enabled
```

Comment:
please add prefix to differentiate which is_enabled is this field belongs to and test once